virttest/utils_misc.py

```diff
@@ -196,6 +196,20 @@ def get_usable_memory_size(align=None):
     return usable_mem
 
 
+def align_value(value_to_be_aligned, align=1024):
+    """
+    Align the 'value_to_be_aligned' to 1024 multiples.
+
+    :param value_to_be_aligned: value to be checked or changed
+                                to 1024 multiples
+    :return: modified input value
+    """
+    value_aligned = int(value_to_be_aligned)
+    if value_aligned % 1000 == 0:
```

Comment:
so this would fail if,
`value_to_be_aligned = 999`

```python
if value_aligned % align != 0:
    value_aligned = ((value_aligned / align) + 1) * align
return value_aligned
```

Reply:
@balamuruhans 
With the above we get a return of '2023', which is wrong.
The following will give the correct answer but still this is not what I wanted. 
```
if value_aligned % align != 0:
    value_aligned = ((value_aligned // align) + 1) * align
return value_aligned
```

Memory value what I wanted is multiples of 1024 and should be exponet of 2.
example: 512000 is multiples of 1024 but,
512000/1024 = 500 (this is not the exponent of 2).
My expectation is 512*1024=524288 instead of 512000.

Reply:
@balamuruhans 
I need to convert the following values to 
```
512000 to 524288
1024000 to 1048576
1536000 to 1572864
2560000 to 2621440
```
This is how libvirt is doing. If I choose multiples of 1024 or the power 2 is not helping.
because:
checking multiples of 1024 and power 2 works for first two values 
i.e. 512000, 1024000
but not for third and fourth values
i.e. 1536000, 2560000

so for all the values
(value/1000)*1024 is working and giving values matching with libvirt.